godel_surface_detection/src/detection/surface_detection.cpp

```diff
@@ -371,8 +339,13 @@ namespace godel_surface_detection
         return false;
 
       // Create Processing Cloud
-      auto process_cloud_ptr = downsampleCloud(*full_cloud_ptr_, random_engine_, DOWNSAMPLE_NUMBER);
+      pcl::PointCloud<pcl::PointXYZRGB>::Ptr process_cloud_ptr (
+            new pcl::PointCloud<pcl::PointXYZRGB>());
       process_cloud_ptr->header = full_cloud_ptr_->header;
+      pcl::VoxelGrid<pcl::PointXYZRGB> vox;
```

Comment:
Any reason we might want to disable the filter and bypass this step? It's easy enough to add later regardless.